roles/container-engine/kata-containers/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+kata_containers_version: 1.11.1
+kata_containers_release_url: https://github.com/kata-containers/runtime/releases/download/{{ kata_containers_version }}/kata-static-{{ kata_containers_version }}-{{ ansible_architecture }}.tar.xz
+kata_containers_dir: /opt/kata
+kata_containers_config_dir: /etc/kata-containers
+kata_containers_containerd_bin_dir: /usr/local/bin
+
+kata_containers_qemu_default_memory: "{{ ansible_memtotal_mb }}"
+kata_containers_qemu_debug: 'false'
+kata_containers_qemu_sandbox_cgroup_only: 'true'
+kata_containers_qemu_enable_mem_prealloc: 'false'
```

Comment:
Are those variables used ?

Reply:
Yes, all of them are used here: _roles/container-engine/kata-containers/templates/configuration-qemu.toml.j2_

Reply:
indeed my bad